rust-runtime/aws-smithy-types/src/lib.rs

```diff
@@ -30,7 +30,7 @@ pub use error::Error;
 /// Binary Blob Type
 ///
 /// Blobs represent protocol-agnostic binary content.
-#[derive(Debug, PartialEq, Clone)]
+#[derive(Debug, PartialEq, Eq, Hash, Clone)]
```

Comment:
If I am not mistaken (@unexge can comment on this), we need to derive the same also on the Python wrappers that are found inside `aws-smithy-http-server-python/src/types.rs`.